packages/manager/modules/billing/src/billing/autoRenew/agreements/details/user-agreements-details.controller.js

```diff
@@ -0,0 +1,89 @@
+import filter from 'lodash/filter';
+import get from 'lodash/get';
+
+export default [
```

Comment:
Maybe we could use `/* @ngInject */` on this function (it requires to rename some parameter `Service` should be renamed `UserAccountServicesAgreements`...)